app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    words = command.split(" ")
+
+    if words[1] == words[2]:
+        return
+
+    with open(words[1], "r") as old_file:
+        with open(words[2], "w") as new_file:
+            content = old_file.read()
+            new_file.write(content)
```

Comment:
![image](https://github.com/mate-academy/py-copy-file/assets/103201010/ce77bc65-04e6-44a5-8919-dabee4df367c)
fix this